packages/babel-plugin-transform-function-sent/test/fixtures/function-sent/after-yield/actual.js

```diff
@@ -0,0 +1,5 @@
+(function* () {
```

Comment:
Does Babel have some functions to check if two nodes are in the same flow branch?

Because the fact that `yield` comes before `function.sent` isn't enough:
```js
(function* (foo) {
  foo && yield;
  function.sent;
}(false));
```